CMSgov/cms-carts-seds
# [CMDCT-3976] <11 Print View Fix

### Description


Certain fields are beings asked to show <11 if a number is given >0 and <= 10. This PR does this by adding a mask property, as well passing the printView prop to the integer component to mask these values. You can see this working here, where computed values will still calculate correctly with the original value but the fields themselves will show <11 

![Screenshot 2024-09-09 at 12 52 49 PM](https://github.com/user-attachments/assets/37234522-9426-44b2-8bf7-b869e5314a74)

**NOTE: Associated ETL PR will follow as this needs to reach production first before it can be used.**

### Related ticket(s)


CMDCT-3976

---
### How to test

Open the 2023 report for AL.
Find a field thats been given the mask (Like 3C's Part 2 question 1-4 (List found at https://jiraent.cms.gov/browse/CMDCT-3976). 
Click the print button at the bottom of the page and click form
See that your answer is <11 if you gave an answer between 1-10!

---
### Pre-review checklist


---
### Pre-merge checklist


#### Review
